model/lifecycle.go

```diff
@@ -373,6 +380,18 @@ func RestartVersion(versionId string, taskIds []string, abortInProgress bool, ca
 		return errors.Wrap(err, "unable to find version")
 	}
 	return errors.Wrap(version.UpdateStatus(evergreen.VersionStarted), "unable to change version status")
+
+}
+
+// RestartVersions restarts completed tasks associated with a set of versionId.
```

Comment:
this comment also will need modifying, since the restart isn't related to the version IDs but rather the set of patches

Reply:
done